kubeflow/fairing/backends/backends.py

```diff
@@ -180,8 +180,10 @@ def get_training_deployer(self, pod_spec_mutators=None):
         pod_spec_mutators.append(aws.add_aws_credentials_if_exists)
         return Job(namespace=self._namespace, pod_spec_mutators=pod_spec_mutators)
 
-    def get_serving_deployer(self, model_class, pod_spec_mutators=None): # pylint:disable=arguments-differ
-        return Serving(model_class, namespace=self._namespace, pod_spec_mutators=pod_spec_mutators)
+    def get_serving_deployer(self, model_class, service_type='ClusterIP', # pylint:disable=arguments-differ
```

Comment:
Are you sure to set the default value to `ClusterIP`, if so the service will not public and only cluster inter can visit that, personally I suggest set to `LoadBalancer`, user can set thet to `ClusterIP` if user do not want to public that, generally the service need be visit from out of cluster.

Reply:
As I think, most of the fairing users develop their model inside jupyter notebook using kubeflow. If that's the majority case, we can set default value to `ClusterIP`, users need to use `Loadbalancer` when they move their development work on laptop. 

Load Balancer will provision resources and that's also limitation at account level. That's the motivation we like to move to `ClusterIP` as default option for aws backend